src/js/veteran-id-card/containers/VeteranIDCard.jsx

```diff
@@ -5,17 +5,32 @@ import RequiredLoginView from '../../common/components/RequiredLoginView';
 import RequiredVeteranView from '../components/RequiredVeteranView';
 import EmailCapture from './EmailCapture';
 
+const rateLimitAuthed = window.settings.vic.rateLimitAuthed;
+const rateLimitUnauthed = window.settings.vic.rateLimitUnauthed;
+
 class VeteranIDCard extends React.Component {
 
   componentDidMount() {
-    if (!this.props.profile.accountType) {
-      window.dataLayer.push({ event: 'vic-unauthenticated' });
+    // Redirect users to email form based on rate limits for unauthed or authed users
+    if (this.props.profile.accountType) {
+      if (this.props.serviceRateLimitedAuthed) {
+        window.dataLayer.push({ event: 'vic-authenticated-ratelimited' });
+        this.renderEmailCapture = true;
```

Comment:
Storing this as a plain old class member instead of state variable because it's not something that changes, but open to arguments for why it should be part of the component state.